Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py

```diff
@@ -2299,6 +2299,9 @@ def __init__(self, howMuch, dataset):
     ('BsToJpsiPhi_mumuKK_14TeV_TuneCP5_cfi', UpgradeFragment(Kby(910,9090),'BsToJpsiPhi_mumuKK_14TeV')), # 1.1%
     ('BuToJPsiPrimeKToJPsiPiPiK_14TeV_TuneCP5_pythia8_cfi', UpgradeFragment(Kby(223,2222),'BuToJPsiPrimeKToJPsiPiPiK_14TeV')), # 5.7%
     ('Psi2SToJPsiPiPi_14TeV_TuneCP5_pythia8_cfi', UpgradeFragment(Kby(45,500),'Psi2SToJPsiPiPi_14TeV')), # 24.6%
+    ('XiMinus_14TeV_SoftQCDInel_TuneCP5_cfi', UpgradeFragment(Kby(9000,90000),'XiMinus_14TeV')), #1.1% (2%)
```

Comment:
Historically, we have added new fragments to the end of the list to avoid changing the workflow numbers for existing fragments. @cms-sw/pdmv-l2 should comment if this policy should still be followed.

Reply:
I think this https://github.com/cms-sw/cmssw/pull/38397 is a mistake. We should preserve the order of the workflow. FYI @kskovpen

Reply:
Hi Adriano, my apologies, let's preserve the order, as was mentioned by @kpedro88 and @srimanob. I keep forgetting about this restriction.

Reply:
I'll fix it for both

Reply:
Thanks Adriano!

Reply:
Should be properly ordered now